.prettierrc.yaml

```diff
@@ -1,2 +1,5 @@
-printWidth: 80
+semi: false
+singleQuote: true
+printWidth: 100
+trailingComma: all
 proseWrap: always
```

Comment:
How do we want to wrap Markdown text?

https://prettier.io/docs/options#prose-wrap

I believe we are already using `proseWrap: always` in all our repositories.

By default, Prettier does not apply any line wrapping rules for Markdown text.

Reply:
My preference: keep `proseWrap: always`.

Long lines are difficult to review on GitHub, as a comment for one line applies to the entire paragraph on that single line.

Reply:
I find diffs very noisy with wrapping, and prefer without wrapping